easybuild/easyconfigs/c/CFITSIO/CFITSIO-4.1.0-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,50 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Denis Kristak
+easyblock = 'CMakeMake'
+
+name = 'CFITSIO'
+version = '4.1.0'
+
+homepage = 'https://heasarc.gsfc.nasa.gov/fitsio/'
+description = """CFITSIO is a library of C and Fortran subroutines for reading and writing data files in
+FITS (Flexible Image Transport System) data format."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/']
+sources = ['%%(namelower)s-%s.tar.gz' % version]
+patches = ['CFITSIO-3.48_install_test_data.patch']
+checksums = [
+    'b367c695d2831958e7166921c3b356d5dfa51b1ecee505b97416ba39d1b6c17a',  # cfitsio-4.1.0.tar.gz
+    'dbf16f857f133468fc1e6a793c6e89fca66d54796593e03606f2722a2a980c0c',  # CFITSIO-3.48_install_test_data.patch
+]
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('CMake', '3.23.1'),
+]
+
+dependencies = [
+    # curl for HTTPS support
+    ('cURL', '7.83.0'),
+    ('zlib', '1.2.12'),
+]
+
+# make would create just static libcfitsio.a. 
+# Let's create dynamic lib and testprog too.
+#buildopts = '&& make shared && make testprog'
```

Comment:
`CFITSIO-4.1.0-GCCcore-11.3.0.eb:36:1`: E265 block comment should start with '# '

```suggestion
# buildopts = '&& make shared && make testprog'
```